Makefile

```diff
@@ -121,21 +122,21 @@ lipo:
 
 android_arm64-v8a:
 	rm -rf build/android-arm64-v8a
-	cmake -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DCMAKE_SYSTEM_NAME=Android -DCMAKE_ANDROID_API=21 -DCMAKE_ANDROID_NDK=/Users/sandrohanea/Library/Developer/Xamarin/android-sdk-macosx/ndk-bundle -S . -B build/android-arm64-v8a
+	cmake $(CMAKE_PARAMETERS) -DCMAKE_ANDROID_ARCH_ABI=arm64-v8a -DCMAKE_SYSTEM_NAME=Android -DCMAKE_ANDROID_API=21 -DCMAKE_ANDROID_NDK=$(NDK_PATH) -S . -B build/android-arm64-v8a
 	cmake --build build/android-arm64-v8a
 	mkdir -p Whisper.net.Runtime/android-arm64-v8a
 	cp build/android-arm64-v8a/whisper.cpp/libwhisper.so Whisper.net.Runtime/android-arm64-v8a/libwhisper.so
 
 android_x86:
 	rm -rf build/android-x86
-	cmake -DCMAKE_ANDROID_ARCH_ABI=x86 -DCMAKE_BUILD_TYPE=$(BUILD_TYPE) -DCMAKE_SYSTEM_NAME=Android -DCMAKE_ANDROID_API=21 -DCMAKE_ANDROID_NDK=/Users/sandrohanea/Library/Developer/Xamarin/android-sdk-macosx/ndk-bundle -S . -B build/android-x86
+	cmake $(CMAKE_PARAMETERS) -DCMAKE_ANDROID_ARCH_ABI=x86 -DCMAKE_SYSTEM_NAME=Android -DCMAKE_ANDROID_API=21 -DCMAKE_ANDROID_NDK=$(NDK_PATH) -S . -B build/android-x86
```

Comment:
Yeah, I replaced it when I released 1.4.0 and forgot to properly fix it in a generic way.

What do you think about something like:

```
if [ -n "$NDK_PATH" ]; then
  NDK="$NDK_PATH"
elif [ -d "$HOME/Library/Developer/Xamarin/android-sdk-macosx/ndk-bundle" ]; then
  NDK="$HOME/Library/Developer/Xamarin/android-sdk-macosx/ndk-bundle"
else
  echo "NDK_PATH not defined and NDK not found at default location on Mac."
  exit 1
fi

```

And then we'll just use $NDK

This way, will be even easier and won't require setting some env variable if VS is installed.

Reply:
Good idea, https://github.com/sandrohanea/whisper.net/pull/34/commits/88dae6b8b94f453f578a9c58280a22a00c7b54ce

What do you think?